api/package.json

```diff
@@ -13,10 +13,10 @@
     "start:prod": "NODE_OPTIONS=\"--max-old-space-size=2048\" NODE_PATH=dist/src node dist/src/main",
     "build": "rm -rf dist && tsc",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
-    "test": "jest --config test/jest-config.json -i",
+    "test": "NODE_OPTIONS=\"--max-old-space-size=4096\" node --expose-gc ./node_modules/.bin/jest --config test/jest-config.json -i --logHeapUsage --detectOpenHandles --forceExit ",
```

Comment:
do we really need all of this?

Reply:
I'd say:
Increasing heap size for node: yes
expose garbage collector: no
call jest from node modules (to avoid some global version running): yes
tracking memory usage: yes (I found it very useful and a secret tools tha
the rest of the flags are helping with mem usage in testing and clearing up some async jobs so I'd say: yes